apps/protocol-frontend/src/hooks/useContributionMetadataStore.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation } from '@tanstack/react-query';
+import { INFURA_PROJECT_ID, INFURA_PROJECT_SECRET } from '../utils/constants';
+import { MintedContributionSchemaV1, IPFS } from '@govrn/protocol-client';
+
+const useContributionMetadataStore = () => {
+  const { mutateAsync, isLoading, isError, isSuccess } = useMutation(
+    ['ContributionMetadataStore'],
+    async (data: Omit<MintedContributionSchemaV1, 'schema'>) => {
```

Comment:
Nice, so when/if we update that Schema we'll have the type safety here

Reply:
yea